lib/main.dart

```diff
@@ -1,125 +1,145 @@
+                            children: [
+                              MiddleText(s: 'Available on ${m.releaseDate}'),
+                              MiddleText(s: m.voteAverage),
+                            ],
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: m.getGenreElements(),
+                        ),
+                        const SizedBox(
+                          height: 20,
+                        ),
+                        Stack(
+                          children: [
+                            Image.network(m.poster),
+                            const Positioned(
+                              top: 15,
+                              right: 15,
+                              child: Column(
+                                children: [
+                                  Icon(
+                                    Icons.favorite_outlined,
+                                    size: 55,
+                                    color: Colors.red,
+                                  ),
+                                  FloatingActionButton(
+                                    onPressed: null,
+                                    child: Icon(Icons.add),
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Positioned(
+                              width: 350,
+                              top: 170,
+                              right: 10,
+                              child: Text(
+                                m.overview,
+                                style: const TextStyle(
+                                  fontWeight: FontWeight.w900,
+                                  fontSize: 20,
+                                  color: Colors.white70,
+                                ),
+                              ),
+                            ),
+                          ],
+                        ),
+                      ],
+                    ),
+                  )),
+            )));
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({super.key, required this.title});
-
-  // This widget is the home page of your application. It is stateful, meaning
-  // that it has a State object (defined below) that contains fields that affect
-  // how it looks.
-
-  // This class is the configuration for the state. It holds the values (in this
-  // case the title) provided by the parent (in this case the App widget) and
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
-
-  final String title;
-
-  @override
-  State<MyHomePage> createState() => _MyHomePageState();
-}
-
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+class MiddleText extends StatelessWidget {
```

Comment:
abstract this class into his own file, as a convention is recommended to have one class per file

Reply:
done